lib/LightPanels/LightPanel.js

```diff
@@ -0,0 +1,200 @@
+/**
+* Contains the declaration for the {@link module:moonstone/LightPanel~LightPanel} kind.
+* @module moonstone/LightPanel
+*/
+
+var
+	kind = require('enyo/kind'),
+	LightPanel = require('enyo/LightPanels').Panel;
+
+var
+	Spotlight = require('spotlight');
+
+var
+	Header = require('../Header');
+
+/**
+* A light-weight panels implementation that has basic support for side-to-side transitions
+* between child components.
+*
+* @class LightPanel
+* @extends module:enyo/Control~Control
+* @ui
+* @public
+*/
+module.exports = kind(
+	/** @lends module:moonstone/LightPanel~LightPanel.prototype */ {
+
+	/**
+	* @private
+	*/
+	name: 'moon.LightPanel',
+
+	/**
+	* @private
+	*/
+	kind: LightPanel,
+
+	/**
+	* @private
+	*/
+	classes: 'moon-light-panel',
```

Comment:
This could be a breaking change (ug) but I think the usual convention is `moon-lightpanel`. We're not consistent in that so if it feels too painful, let's leave it.